### PR TITLE
MRG, DOC: Update links from linkcheck

### PR DESCRIPTION
**Pull request by larsoner**

### CODE REVIEW

File: doc/changes/names.inc [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 .. _Andrew Dykstra: https://github.com/adykstra
 
-.. _Romain Trachel: http://www.lscp.net/braware/trachelBr.html
+.. _Romain Trachel: https://fr.linkedin.com/in/trachelr
```
Comment (agramfort) [inline doc/changes/names.inc:25]:
```suggestion
.. _Romain Trachel: https://github.com/trachelr
```

Comment (larsoner) [reply doc/changes/names.inc:25]: I used the URL in his GitHub profile

Comment (larsoner) [reply doc/changes/names.inc:25]: (to me if someone has it set, it's the URL they want you looking at to get information on them)

**Comment by agramfort** [comment]
fair enough

>

**Pull request merged**